phpunit.xml

```diff
@@ -18,6 +18,7 @@
             <directory suffix=".php">src</directory>
             <exclude>
                 <directory>src/Protobuf</directory>
+                <directory>src/DocScan/Session/Retrieve/Contracts</directory>
```

Comment:
Generally we take the approach of only excluding generated files, so can we remove this line? 
```suggestion
```

Reply:
Yes